include/swift/Driver/Driver.h

```diff
@@ -128,6 +128,9 @@ class OutputInfo {
   /// DWARF output format version number.
   std::optional<uint8_t> DWARFVersion;
 
+  /// Emit debug info as split-DWARF (Fission) in separate .dwo files.
+  bool DWARFFission = false;
```

Comment:
Let's stick with either calling everything "Split" or "DWO". I think post DWARF 5, the "fission" name is less often used and could confuse people.

Reply:
OK. I will remove the "fission" in the names. @compnerd Please let me know if you are not on the same page.

Reply:
Sure, I think that `SplitDWARF` is fine to use as an alternative to `DWARFFission`. We should be able to use the former throughout.

Reply:
Done